test/fixtures/linter/rules/NoDeprecatedApi/testsuite.qunit.js

```diff
@@ -0,0 +1,34 @@
+sap.ui.define(function () {
+	"use strict";
+	return {
+		name: "QUnit test suite with deprecated themes (JS)",
+		defaults: {
```

Comment:
Please, consider that any key in the structure could be quoted.
For example, if you put quotes on `defaiults` or `ui5` properties, the check would stop working.

```js
sap.ui.define(function () {
	"use strict";
	return {
		name: "QUnit test suite with deprecated themes (JS)",
		"defaults": {
			page: "ui5://test-resources/sap/ui/demo/todo/Test.qunit.html?testsuite={suite}&test={name}",
			qunit: {
				version: 2
			},
			sinon: {
				version: 4
			},
			ui5: {
				language: "EN",
				theme: "sap_bluecrystal" // positive finding
			}
		},
		tests: {
			"unit/unitTests": {
				theme: "sap_belize", // negative finding (wrong place)
				title: "Unit tests for Todo App",
				"ui5": {
					theme: "sap_belize_plus", // positive finding
				}
			},
			"integration/opaTests": {
				title: "Integration tests for Todo App",
				ui5: {
					"theme": "sap_belize_hcb", // positive finding
				}
			}
		}
	};
});
```

Reply:
Done